src/dotty/tools/dotc/transform/TreeTransform.scala

```diff
@@ -174,32 +174,22 @@ object TreeTransforms {
  }
 
   /** A helper trait to transform annotations on MemberDefs */
-  trait AnnotationTransformer extends MiniPhaseTransform with InfoTransformer {
+  trait AnnotationTransformer extends MiniPhaseTransform with DenotTransformer {
```

Comment:
Why is this change needed?
As annotation transformers also change types, isn't every annotation transformer and info-transformer?